src/marten/cli/manage/command/new.cr

```diff
@@ -30,6 +31,12 @@ module Marten
             on_argument(:name, "Name of the project or app to initialize") { |v| @name = v }
             on_option_with_arg(:d, :dir, arg: "dir", description: "Optional destination directory") { |v| @dir = v }
             on_option(:"with-auth", "Adds authentication to newly created projects") { @with_auth = true }
+            on_option_with_arg(
+              :database,
```

Comment:
Can we also provide a short flag alternative to this new option (eg. `-db`)? 

```suggestion
            on_option_with_arg(
              :db,
              :database,
```

Reply:
I've tried it, but apparently the crystal `OptionParser` allows only one character for the short flag. 

If `:db` is defined as a short flag and a new project is created with `marten new -db=mysql project dummy_project` then a project `b=mysql` will be created (because `-d` is used with the value `b=mysql`)

Reply:
Sounds good. Let's keep it that way!